modules/openapi-generator/src/main/resources/typescript/configuration.mustache

```diff
@@ -12,16 +12,16 @@ import { BaseServerConfiguration, server1 } from "./servers{{importFileExtension
 import { configureAuthMethods, AuthMethods, AuthMethodsConfiguration } from "./auth/auth{{importFileExtension}}";
 
 export interface Configuration<M = Middleware> {
-  readonly baseServer: BaseServerConfiguration;
-  readonly httpApi: HttpLibrary;
-  readonly middleware: M[];
-  readonly authMethods: AuthMethods;
+    readonly baseServer: BaseServerConfiguration;
+    readonly httpApi: HttpLibrary;
+    readonly middleware: M[];
+    readonly authMethods: AuthMethods;
 }
 
 // Additional option specific to middleware merge strategy
 export interface MiddlewareMergeOptions {
-  // default is `'replace'` for backwards compatibility
-  middlewareMergeStrategy?: 'replace' | 'append' | 'prepend';
+    // default is `"replace"` for backwards compatibility
```

Comment:
revert these to reduce diff?

Reply:
I can understand that is it better to have smaller PRs, but cleaning up the code format is also useful.
On the other hand I would not want to create a separate PR just for formatting changes (right?).
So I think keeping them here as a dedicated commit still makes the most sense to me.

Reply:
> On the other hand I would not want to create a separate PR just for formatting changes (right?).

I think that's exactly what we should do - separate meaningful changes like this in here (refactor) from cosmetic only (style). It makes it easy to revert the chunk that causes trouble, if there is an issue (the refactor potentilly) and leave the things that are non-contentious and not risky (style) in.
A style-change-only PR is also easy to review and merge. And the noise for review is less as well, which means both PRs separately can be reviewed much faster than one big one.

> So I think keeping them here as a dedicated commit still makes the most sense to me.

see above

Reply:
Ok, I understand. I'll try to further split the changes over more PRs.